host/inc/getparameters.h

```diff
@@ -96,6 +96,7 @@ typedef struct
 		float adam_beta1;
 		float adam_beta2;
 		float adam_epsilon;
+		int no_output_xml;
```

Comment:
As the command line parameter is to output xml, it's better to not invert this inside the code: let's do `bool output_xml` instead